README.md

```diff
@@ -36,8 +36,8 @@ And the following functionalities:
   
     1. Add an event listener to the form with `.addEventListener`. What event do you want to listen for?
     2. Remember, what does `event.preventDefault()` do?
-    3. Grab the value the user typed from the text input. Do you remember what property of the input node has this?. If not Google it or ask a peer.
-    4. Create new `li` element with `document.createElement()`. Set its `innerText` property to be the text the user typed.
+    3. Grab the value the user typed from the text input. Do you remember what property of the input node has this? If not Google it or ask a peer.
+    4. Create new `li` element with `document.createElement()`. Set its `textContent` property to be the text the user typed.
```

Comment:
If you don't mind me being a stickler: 

* https://developer.mozilla.org/en-US/docs/Web/API/Node/textContent#differences_from_innertext
* https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext

Reply:
Thank you! We'll have to cross reference this against whatever we have been teaching but for sure we always appreciate getting the most correct content in here.

Reply:
And this is perfect and we are teaching `innerText` everywhere else anyway :)

Reply:
@mikeboyle just to be clear, I'm suggesting switching everything to `textContent` as that's the more performance-friendly thing (since it doesn't require a reflow)... but since `innerText` is everywhere else I'll just switch back to `innerText`.

Reply:
oh so sorry I totally misread that. Can you switch it back for now and then I / my team will do a sweep of everything we're teaching and see if we can update it in time.